include/rocksdb/file_checksum.h

```diff
@@ -50,6 +54,7 @@ class FileChecksumGenerator {
   virtual std::string GetChecksum() const = 0;
 
   // Returns a name that identifies the current file checksum function.
+  // The name should not contain ':'.
```

Comment:
Why?

Reply:
This is leftover from an earlier draft :P (will remove it)